tests/Database/Oci8SchemaGrammarTest.php

```diff
@@ -487,6 +487,17 @@ public function testAddingIndex()
         $this->assertEquals('create index baz on users ( foo, bar )', $statements[0]);
     }
 
+    public function testAddingFullTextIndex()
```

Comment:
SQL generated is as expected but doesn't work when executed on actual DB migration.

Reply:
omg.. multiple columns index in oracle is implemented with ctx_ddl preferences 😅

Reply:
Sorry, ctx_ddl is something new to me 😅. Do I need some special setup for this work? 

Will also try to research this. Thanks!

Reply:
I fixed this bug and implemented ctx_ddl preferences to create multi_columns_index oracle feature. In method compileFullText(), i implemented one index to each column, enabling the search of multi column preference. If you have any suggestions, don't hesitate to help me 😅. Thanks!

Reply:
Hey @hpacleb, you have a new suggestions? 😁